server/src/test/java/org/opensearch/action/support/master/TransportMasterNodeActionTests.java

```diff
@@ -439,7 +439,9 @@ public void testDelegateToFailingMaster() throws ExecutionException, Interrupted
         if (rejoinSameMaster) {
```

Comment:
Method name, `testDelegateToFailingMaster` -> `testDelegateToFilingClusterManager` and the above `rejoinSameMaster` -> `rejoinSameClusterManager`

Reply:
Changed the method and variable name. 👍